WordPress/src/main/java/org/wordpress/android/util/WPMediaUtils.java

```diff
@@ -151,4 +153,51 @@ public void onCancel(DialogInterface dialog) {
         // Do not ask again
         AppPrefs.setImageOptimizePromoRequired(false);
     }
+
+    /**
+     * Given a media error returns the error message to display on the UI.
+     *
+     * @param error The media error occurred
+     * @return String  The associated error message.
+     */
+    public static String getErrorMessage(final Context context, boolean suggestMediaOptimization, final MediaModel media, final MediaStore.MediaError error) {
```

Comment:
That logic looks incorrect, though. For example, [here](https://github.com/wordpress-mobile/WordPress-Android/pull/6285/commits/3316c171fcb2b63ee934050ebf06eed0dbc278eb#diff-fdc9b5c706ac889526b8df93c55351ceR635) we're passing whether image optimization is enabled, so we're only going to suggest optimization if it's already enabled. Or am I reading that wrong?